ci/jenkins/test.sh

```diff
@@ -353,17 +353,16 @@ function deliver_antrea_windows {
             sleep 5
             ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "W32tm /resync /force" | grep successfully && break
         done
+        # Avoid potential resync delay error
+        sleep 5
```

Comment:
Could you explain what would happen without this?

Reply:
I found sometimes the first pulling action failed after windows time sync. And the failed reason is still the wrong time. So I think it should caused by date sync delay on windows. I didn't seen the same error with this setup.